examples/demo-project/backend/garden.yml

```
@@ -9,6 +9,10 @@ variables:
 
 services:
   - name: backend
+    localMode:
+      localAppPort: 8090
+      command: [] # example: [ "npm", "run", "serve", "-p", "${var.localAppPort}" ]
```

in this case I think it should be
```suggestion
      command: ["go", "run", "main.go"] # example: [ "npm", "run", "serve", "-p", "${var.localAppPort}" ]
```

---

Makes sense! We can also create a separate example like `demo-project-local-mode`, wdyt?

---

I updated the documentation, CLI help and this file too. In general, the `command` should not be tied to the current service/module directory. It should be executable from any location because a local service can reside in an arbitrary file system location defined by a user. I added some clarifying comments.